openc3-cosmos-init/plugins/packages/openc3-cosmos-tool-scriptrunner/src/tools/ScriptRunner/ScriptRunner.vue

```diff
@@ -900,6 +900,9 @@ export default {
       if (role == 'admin' || role == 'operator') {
         this.readOnlyUser = false
         this.executeUser = true
+      } else if (role == 'runner') {
+        this.readOnlyUser = true
```

Comment:
This line isn't needed and could cause problems if a user had both operater and runner.